build.sbt

```diff
@@ -1,13 +1,13 @@
 import sbtassembly.AssemblyKeys.assembly
 import sbt._
 
+scalaVersion := "2.11.12"
```

Comment:
Not sure if this is actually required, once I removed the transitive plugins and sbt-auto-build it only seemed to compile 2.10 - so added this